Source: written by Vincent Untz <vuntz@gnome.org>
Upstream: yes, Gnome Bug #603690
Reason: Install DBusGLib-1.0.gir if dbus is enabled
Index: gir-repository-0.6.5/gir/Makefile.am
===================================================================
--- gir-repository-0.6.5.orig/gir/Makefile.am
+++ gir-repository-0.6.5/gir/Makefile.am
@@ -12,8 +12,7 @@ CUSTOM_GIRSOURCES += $(AVAHI_GIRSOURCES)
 endif
 EXTRA_DIST += $(AVAHI_GIRSOURCES)
 
-MISSIONCONTROL_GIRSOURCES = DBusGLib-1.0.gir \
-			    TelepathyGLib.gir \
+MISSIONCONTROL_GIRSOURCES = TelepathyGLib.gir \
 			    LibTelepathy.gir
 if BUILD_MISSIONCONTROL
 CUSTOM_GIRSOURCES += $(MISSIONCONTROL_GIRSOURCES)
@@ -29,6 +28,8 @@ libgirepo_DBus_custom_la_SOURCES = 	\
 EXTRA_DIST += $(srcdir)/dbus.c $(srcdir)/dbus.h
 
 if BUILD_DBUS
+BUILT_GIRSOURCES += DBusGLib-1.0.gir
+
 DBus-1.0.gir: libgirepo-DBus-custom.la
 	$(INTROSPECTION_SCANNER) -v --namespace DBus --nsversion=1.0 \
 	    --add-include-path=$(srcdir) --add-include-path=. \
